app/models/user.rb

```diff
@@ -114,4 +115,13 @@ def revoke_all
   def unread_dm_between(user_id)
     DirectMessage.where('user_id = ? and to_user_id = ?', user_id, id).unread
   end
+
+  def wechat_enable
+    authorizations.any? {|auth| auth.provider == "wechat" }
+  end
+
+  def weibo_enable
+    authorizations.any? {|auth| auth.provider == "weibo" }
+  end
```

Comment:
when you name an identifier with a verb, you imply it is an action. so you should use `wechat_enabled` and `weibo_enabled` here instead.